runtime/compiler/env/J9PersistentInfo.hpp

```diff
@@ -169,6 +169,7 @@ class PersistentInfo : public OMR::PersistentInfoConnector
          _requireJITServer(false),
          _localSyncCompiles(true),
          _JITServerAOTCacheName(),
+         _JITServerAOTMaxBytes(300000000), // 300M
```

Comment:
I suggest using something like 300*1024*1024 to avoid counting zeros and prevent errors.